Rakefile

```diff
@@ -234,21 +236,11 @@ namespace :git do
   end
 end
 
-desc "Open the project in Xcode"
-task :xcode => [:dependencies] do
+desc 'Open the project in Xcode'
+task xcode: [:dependencies] do
   sh "open #{XCODE_WORKSPACE}"
 end
 
-def fold(label, &block)
-  puts "travis_fold:start:#{label}" if is_travis?
-  yield
-  puts "travis_fold:end:#{label}" if is_travis?
-end
-
-def is_travis?
-  return ENV["TRAVIS"] != nil
-end
-
```

Comment:
I noticed these references to TravisCI and figured it would be safe to remove them.

Reply:
Re-added in https://github.com/Automattic/simplenote-ios/commit/57711bfcb08741d7d9abf5dc2253dac1959abb17